lib/control/v1/signature.js

```diff
@@ -21,7 +21,7 @@ class Server {
   }
 
   start() {
-    this.process = CP.spawn(this.path, ['-p', this.port]);
+    this.process = bundledGemSpawn(this.path, ['-p', this.port]);
```

Comment:
Thanks, I forgot to call the cwd command.
And I changed the bundle package back to bundle install

Both changes added in e388326.